backend/open_webui/routers/audio.py

```
@@ -818,8 +845,190 @@ def transcription_handler(request, file_path, metadata):
+                            "content": [
+                                {
+                                    "type": "input_audio",
+                                    "input_audio": audio_base64,
+                                },
+                                {"type": "text", "text": text_instruction},
+                            ],
+                        }
+                    ],
+                }
+
+                r = requests.post(
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)